content/magnetics/magnetics_susceptibility.rst

```diff
@@ -166,6 +166,12 @@ Organically derived susceptibility
 Organic chemistry can be important in understanding magnetic survey results for petroleum exploration applications. Under certain conditions magnetic anomalies caused by organic chemical activity can provide indicators of petroleum reservoir in underlying rocks. This is described in a supplementary article (Stone et al, 2004).
 
 
+Field measurements of magnetic susceptibility using a KT10
+----------------------------------------------------------
+
+.. figure:: 
+	../physical_properties/images/magnetic_susceptibility_measurement_kt10.jpg
```

Comment:
That's weird. It displays well when I run the local version.

I pushed a new version of the susceptibility.rst file (a space before
..figure:: as it seems it worked for the hyperlink this after. I also
respected carefully capital letters). Hopefully it will worked for the
internet version.

2015-09-08 17:04 GMT-07:00 Lindsey <notifications@github.com>:

> In content/magnetics/magnetics_susceptibility.rst
> <https://github.com/ubcgif/geoscixyz/pull/6#discussion_r38994993>:
>
> > @@ -166,6 +166,12 @@ Organically derived susceptibility
> >  Organic chemistry can be important in understanding magnetic survey results for petroleum exploration applications. Under certain conditions magnetic anomalies caused by organic chemical activity can provide indicators of petroleum reservoir in underlying rocks. This is described in a supplementary article (Stone et al, 2004).
> >
> >
> > +Field measurements of magnetic susceptibility using a KT10
> > +----------------------------------------------------------
> > +
> > +.. figure::
> > +	../physical_properties/images/magnetic_susceptibility_measurement_kt10.jpg
>
> It looks like this image is not showing up, is the name different? (
> http://gpg.geosci.xyz/en/latest/content/magnetics/magnetics_susceptibility.html#field-measurements-of-magnetic-susceptibility-using-a-kt10)
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubcgif/geoscixyz/pull/6/files#r38994993>.
>